### DISCUSSION TITLE
Adobe Air + Bulkr White Box Issue

### DISCUSSION THREAD

**Discussion by Trixxx13** [airsdk/Adobe-Runtime-Support#3579] [General]
Hi all,

I can't work out the problem here so hoping someone more knowledgable than me can help.

I recently downloaded Bulkr (and paid for the full version) which requires I download Adobe Air from the Harman website. I have done so and tried multiple times and I end up with the same issue everytime. When I load Bulkr I have a White box show permanently on my screen. Anyone got any ideas what is causing this? I can't work out if its an Adobe AIr problem or a Bulkr Problem. I am using a Macbook Pro (M1 Chip) with the latest OSX, I have tried it on an older machine with an older OSX and get the same issue. Any thoughts would be massively appreciated. Sadly BUlkr customer service is no longer operating it seems.

Thanks i Advance!

<img width="1711" alt="Screenshot 2024-11-19 at 13 18 56" src="https://github.com/user-attachments/assets/82cba625-93fb-4f96-97a6-d1e17113addf">

**Comment by ajwfrost** [comment]
>  which requires I download Adobe Air from the Harman website

Which one did you download? It's very possible they're trying to use a feature in AIR that we had to take out because it was legacy and didn't support Apple Silicon.. you can perhaps install the older version of the AIR runtime which then runs everything in the Rosetta2 mechanism?

So:
1) Uninstall AIR - from /Applications, run the "Adobe AIR Installer" app, which will then guide you though an uninstall.
2) Try installing the 33.1.1.744 version which is further down on the 'runtimes' page: https://airsdk.harman.com/assets/downloads/33.1.1.744/AdobeAIR.dmg

I think you can keep the Bulkr app installed throughout this. It should already be running in Rosetta2 mode I suspect, but the older webkit library is only available in that older runtime.

But then it's also possible that the application there is actually trying to display a web page, and even if you have the webkit component installed, if the web page has been updated separately, it may no longer support the earlier webkit/browser version? I don't suppose you can right-click on the white area and see if there is a context menu, with any information available such as properties or details etc?

thanks

**Comment by Trixxx13** [comment]
That worked. I have so much gratefulness for you right now!!! I did exactly this. For anyone else who has this issue. I did the following things...

In Terminal I did the following things...

Updated to Rosetta
softwareupdate --install-rosetta --agree-to-license

Install the older version of Adobe Air (as ajwfrost suggested)
https://airsdk.harman.com/assets/downloads/33.1.1.744/AdobeAIR.dmg

I followed the install steps on Bulkr (setting the clock to 30/12/2020)

Then I was encountering issues actually loading the sotware

So you find the route to the Bulkr app (right click on the app and click 'copy path')

Then In Terminal I ran the following commands

arch -x86_64 /Applications/Bulkr/Bulkr.app/Contents/MacOS/Bulkr

or this...

arch -x86_64 /Applications/Bulkr/Bulkr.app/Contents/MacOS/Bulkr --debug

Then Viola! it finally worked. Thank you ajwfrost!!!
